aggregator_test.go

```diff
@@ -459,6 +459,134 @@ func TestPerformMigrationsTooManyArgs(t *testing.T) {
 	os.Args = oldArgs
 }
 
+// TestHandleCommandStartService checks whether "start-service"
+// command successfully finishes
+func TestHandleCommandStartService(t *testing.T) {
+	*main.AutoMigratePtr = true
+
+	os.Clearenv()
+	mustLoadConfiguration("./tests/tests")
+	oldArgs := os.Args
+	os.Args = []string{os.Args[0], "start-service"}
+
+	go func() {
+		main.Main()
+	}()
+
+	errCode := main.StopService()
+	assert.Equal(t, main.ExitStatusOK, errCode)
+
+	os.Args = oldArgs
+
+	*main.AutoMigratePtr = false
+}
+
+// TestHandleCommandHelp checks whether "help", "print-help"
+// and undefined command (defaults to help) successfully finish
+func TestHandleCommandHelp(t *testing.T) {
```

Comment:
`TestHandleCommandHelp`, `TestHandleCommandConfig`, `TestHandleCommandEnv`, `TestHandleCommandEnv` and  `TestHandleCommandVersion` are basically doing the check. I would group into a single parametrized terst. Or is there any advantage of doing like this?